climpred/metrics.py

```diff
@@ -21,7 +21,9 @@
     threshold_brier_score,
 )
 
-from .constants import CLIMPRED_DIMS
+# from .constants import CLIMPRED_DIMS
+# TODO: I dont understand why I get an error when I import this, maybe cyclic imports?
```

Comment:
Remove this and above. Maybe add a comment clarifying the import issues for future developers.